tests/protocol/mockwiredata.py

```diff
@@ -38,8 +38,8 @@
         "test_ext": "ext/sample_ext-1.3.0.zip",
         "remote_access": None,
         "in_vm_artifacts_profile": None,
-        "vm_settings": "hostgaplugin/vm_settings.json",
-        "ETag": "1"
+        "vm_settings": None,
+        "ETag": None
```

Comment:
A large number of unit tests use the default DATA_FILE, which does not have a vm_settings.json that matches ext_conf.xml; all those test would hit the retry logic with the sleep before falling back to extensions config.

We will need matching files for all these data files, but for the moment I made DATA_FILE return 404 for vmSettings (falling back to extensions config without hitting the sleep in the retry)